tests/selenium/spec/quickstarts-spec.js

```diff
@@ -47,6 +47,149 @@ describe('quickstarts page default selections spec', () => {
     ]);
   }));
 
+  it('selects okta-sign-in-page + nodejs + express if invalid client in hash', util.itHelper(async () => {
+    await quickstartsPage.navigate('/quickstart/#/undefined/nodejs/express');
+    await quickstartsPage.refresh();
+    expect(await quickstartsPage.getCurrentURL())
+      .to.contain(QuickStartsPage.getSigninPageNodeExpressFragment());
+    expect(await quickstartsPage.getActiveLinkText())
+      .to.include.members([
+      'Okta Sign-In Page',
+      'Node JS',
+      'Express.js'
+    ]);
+  }));
+
+  it('selects okta-sign-in-page + nodejs + express if invalid server in hash', util.itHelper(async () => {
+    await quickstartsPage.navigate('/quickstart/#/okta-sign-in-page/undefined/express');
+    await quickstartsPage.refresh();
+    expect(await quickstartsPage.getCurrentURL())
+      .to.contain(QuickStartsPage.getSigninPageNodeExpressFragment());
+    expect(await quickstartsPage.getActiveLinkText())
+      .to.include.members([
+      'Okta Sign-In Page',
+      'Node JS',
+      'Express.js'
+    ]);
+  }));
+
+  it('selects okta-sign-in-page + nodejs + express if invalid server framework in hash', util.itHelper(async () => {
+    await quickstartsPage.navigate('/quickstart/#/okta-sign-in-page/nodejs/null');
```

Comment:
`/null`?   Really?  Weird.

Reply:
Yup... this is what was coming up in the url which caused this test!  fun times :)